vw_golf_mk4.dbc

```diff
@@ -1433,19 +1433,51 @@ BO_ 2000 Diagnose_1: 8 XXX
  SG_ DI1_KM_Stand_alt : 62|1@1+ (1,0) [0|1] "" XXX
  SG_ DI1_Zeit_alt : 63|1@1+ (1,0) [0|1] "" XXX
 
+CM_ SG_ 80 Checksumme_Airbag_1 "Checksum Airbag_1"
```

Comment:
Can you make sure add semicolons and remove the newlines?

Reply:
@sshane do you mean https://github.com/commaai/opendbc/pull/654/commits/984a9ab3668812d494b3ad9e8c02ae47e8f8815a when you request new lines to be removed? Seems to be a negative value but it's your call.

@sshane please clarify what semicolors are you referring to. I am a bit confused if there is some pattern/style/convention I am missing or if that's a random subjective preference?

Reply:
The [DBC specification](http://mcu.so/Microcontroller/Automotive/DBC_File_Format_Documentation.pdf) calls for a trailing semicolon on the comment lines:

```suggestion
CM_ SG_ 80 Checksumme_Airbag_1 "Checksum Airbag_1";
```

Reply:
@rusefillc we generally don't have strict stylistic guidelines. it's up to the author to make their code tasteful, and we'll review as needed. in this case, I think you were right with the newlines. @sshane why did you want them removed?